scanner/src/test/kotlin/experimental/ExperimentalScannerTest.kt

```diff
@@ -636,7 +639,7 @@ private class FakePackageScannerWrapper(
 ) : PackageScannerWrapper {
     override val details = ScannerDetails(name, "1.0.0", "config")
     override val name = details.name
-    override val criteria: ScannerCriteria? = ScannerCriteria.forDetails(details)
+    override val criteria = ScannerCriteria.forDetails(details)
```

Comment:
I find the type declaration a lot less annoying than the cast and suppress statement in the test code.

Reply:
Yeah, at first I thought it'd just work without the cast or suppress statement, but then I had to add the cast, and then also the suppress statement. While the code proposed here does not trigger any inspection hints anymore, I agree it's a bit clumsy.

I just tested and I can suppress the original hint instead. Will do that.